shared/constants.go

```diff
@@ -21,6 +21,12 @@ const PlatformConfig = BraveHome + "/config.yml"
 // ImageStore ..
 const ImageStore = BraveHome + "/images/"
```

Comment:
for these paths, can we have `filepath.Join` to make it more os-agnostic

Reply:
Unfortunately filepath.Join does not work with constants.

However, Windows is pretty good about accepting forward slashes in paths nowadays - many paths bravetools currently creates are actually a mixture of forward and backwards slashes and they still work fine.